gallery/webpack.config.js

```diff
@@ -16,8 +16,15 @@ module.exports = function(env = { production: false }) {
     new CopyModulesPlugin({
       destination: 'webpack-modules',
       includePackageJsons: true
+    }),
+    new webpack.DefinePlugin({
+      'process.env.PRODUCTION': JSON.stringify(true)
     })
-  ] : [];
+  ] : [
+    new webpack.DefinePlugin({
+      'process.env.PRODUCTION': JSON.stringify(false)
```

Comment:
From https://webpack.js.org/guides/production/ :
> Since webpack v4, specifying [mode](https://webpack.js.org/configuration/mode/) automatically configures [DefinePlugin](https://webpack.js.org/plugins/define-plugin) for you:

Reply:
> tomatically configures [DefinePlugin](https://webpack.js.org/plugins/define-plugin) for you:

Thanks! fixed here: 6934bd3e8e37d07275c29148132f54d7299a3ab8